packages/trilinoscouplings/examples/scaling/Maxwell.xml

```diff
@@ -57,6 +57,7 @@
       <Parameter name="number of equations" type="int" value="3"/>
       <Parameter name="aggregation: type" type="string" value="uncoupled"/>
       <Parameter name="aggregation: drop tol" type="double" value="0.01"/>
+      <Parameter name="tentative: calculate qr" type="bool" value="false"/>
```

Comment:
Why is this needed?

Reply:
That is a remnant of rebasing. Not my change.